distribution/pom.xml

```diff
@@ -171,7 +171,6 @@
 
     <modules>
         <module>fully-loaded</module>
```

Comment:
You're right!

I kept it for elasticsearch java client users because they have to do all those imports by themselves. 

That said, I think (but might be wrong) that in elasticsearch core project we should not add any `optional` dependency. I mean that if people want to exclude a lib, then they can do it with `<exclude>` maven tag. WDYT?

But that's another story and should come in another PR. For now, I'll remove this fully loaded module.